### DISCUSSION TITLE
archive.txt vs done.txt

### DISCUSSION THREAD

**Discussion by kmf** [todotxt/todo.txt#90] [Ideas]
Just wanted to know what is the standard?

I'm seeing different implementations using different file names. Is it part of a published standard?

**Comment by hyperupcall** [comment]
@kmf

The default [todo.cfg](https://github.com/todotxt/todo.txt-cli/blob/fd753c6c2e8e661d9882eb41af4ec3a2429b6484/todo.cfg#L8) comes shipped with three defaults:

```bash
export TODO_FILE="$TODO_DIR/todo.txt"
export DONE_FILE="$TODO_DIR/done.txt"
export REPORT_FILE="$TODO_DIR/report.txt"
```

My guess is they have customized their `DONE_FILE` variable. Of the detailed information at [todotxt/todo.txt](https://github.com/todotxt/todo.txt), I could not find any specific mention for other file names. I personally haven't come across `archive.txt` so I wouldn't know if it's a part of a different "convention" of sorts.

**Reply by kmf** [reply]
Thanks for the answer @hyperupcall 

References to `archive.txt`
- https://www.txdx.eu/getting-started/#the-todotxt-and-archivetxt-files
- https://github.com/benjamonnguyen/obsidian-todotxt-codeblocks
- Some old vim plugins.
